### ISSUE TITLE
Coode looks outdated

### ISSUE THREAD

**Issue by HazemKhaled** [Magento2Translations/Compile#2]
I think you need to push code updates, download command support new branches structure while package and translate no

Also in m2t.php some paths wrong or missing

I think all you need to push latest updates

Thanks

**Comment by MrGekko** [comment]
Yep I'm sorry! that is still a todo. I've updated the internal code to mirror the new branches. But the open compiler is still a wip...
